pkg/accounting/db.go

```
@@ -35,5 +42,20 @@ func NewDb(databaseURL string) (*dbx.DB, error) {
 		_ = db.Close()
```

Don't ignore this error.